M2/Macaulay2/m2/CMakeLists.txt

```diff
@@ -24,8 +24,6 @@ add_custom_command(OUTPUT ${CORE_DIR}/tvalues.m2
   DEPENDS M2-binary
   VERBATIM)
 
-configure_file(version.m2.in ${M2_DIST_PREFIX}/${M2_INSTALL_DATADIR}/Core/version.m2 @ONLY)
```

Comment:
A solution for #3250 would involve reviving this file, so it would be counterproductive to remove it from the build system _now_!